src/style.css

```diff
@@ -1,3 +1,146 @@
-body {
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 14px;
+}
+
+ul,
+li {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+body,
+p {
   margin: 0;
 }
+
+.card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: block;
+  max-width: 160px;
+  max-height: 134px;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.card__text {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  letter-spacing: 0;
+  color: #060b35;
+}
+
+.card__text--y {
+  margin-top: 40px;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  letter-spacing: 0;
+  color: #616070;
+}
+
+.stars--y {
+  margin: 16px 0 24px;
+}
+
+.stars__wrap {
+  display: flex;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stars__star {
+  display: block;
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars__star:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars__number {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  letter-spacing: 0;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__price--y {
+  margin-bottom: 16px;
+}
+
+.card__price-title {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  letter-spacing: 0;
+  color: #616070;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)